lib/command/src/Obelisk/Command/Deploy.hs

```diff
@@ -150,13 +159,88 @@ deployPush deployPath getNixBuilders = do
 deployUpdate :: MonadObelisk m => FilePath -> m ()
 deployUpdate deployPath = updateThunkToLatest $ deployPath </> "src"
 
+keytoolToAndroidConfig :: KeytoolConfig -> Map Text NValue
+keytoolToAndroidConfig conf = Map.fromList
+  [ ("storeFile", NValue_Path $ _keytoolConfig_keystore conf)
+  , ("storePassword", NValue_Text $ T.pack $ _keytoolConfig_storepass conf)
+  , ("keyAlias", NValue_Text $ T.pack $ _keytoolConfig_alias conf)
+  , ("keyPassword", NValue_Text $ T.pack $ _keytoolConfig_keypass conf)
+  ]
+
+
 deployMobile :: MonadObelisk m => String -> [String] -> m ()
 deployMobile platform mobileArgs = withProjectRoot "." $ \root -> do
   let srcDir = root </> "src"
   exists <- liftIO $ doesDirectoryExist srcDir
   unless exists $ failWith "ob test should be run inside of a deploy directory"
-  result <- nixBuildAttrWithCache srcDir $ platform <> ".frontend"
-  callProcessAndLogOutput (Notice, Error) $ proc (result </> "bin" </> "deploy") mobileArgs
+  when (platform == "android") $ do
+    let keystorePath = root </> "android_keystore.jks"
+        keytoolConfPath = root </> "android_keytool_config.json"
+    hasKeystore <- liftIO $ doesFileExist keystorePath
+    when (not hasKeystore) $ do
+      -- TODO log instructions for how to modify the keystore
+      putLog Notice $ "Creating keystore: " <> T.pack keystorePath
+      putLog Notice "Enter a keystore password: "
+      keyStorePassword <- liftIO $ withEcho False getLine
+      putLog Notice "Re-enter the keystore password: "
+      keyStorePassword' <- liftIO $ withEcho False getLine
+      unless (keyStorePassword' == keyStorePassword) $ failWith "passwords do not match"
+      let keyToolConf = KeytoolConfig
+            { _keytoolConfig_keystore = keystorePath
+            , _keytoolConfig_alias = "obelisk"
+            , _keytoolConfig_storepass = keyStorePassword
+            , _keytoolConfig_keypass = keyStorePassword
+            , _keytoolConfig_dname = "CN=example.com, OU=engineering, O=obelisk, L=LinuxLand, S=NY, C=US" -- TODO Read these from config?
```

Comment:
We need to use real data here.

Reply:
going to drop this default `Distinguished Name` configuration and instead let `keytool` prompt for this